examples/data-objects/inventory-app/src/view/counter.tsx

```diff
@@ -3,35 +3,32 @@
  * Licensed under the MIT License.
  */
 
-import { useTreeContext } from "@fluid-experimental/tree-react-api";
-import { SchemaBuilder, TypedField, leaf } from "@fluid-experimental/tree2";
 import * as React from "react";
 
-const schema = SchemaBuilder.required(leaf.number);
-
 export interface ICounterProps {
 	title: string;
-	count: TypedField<typeof schema>;
+	count: number;
+	onIncrement: () => void;
+	onDecrement: () => void;
```

Comment:
To avoid exposing fields in the API, need to revert to previous pattern of 'data in' / 'events out'.

(Ditto for changes to Counter below.)